test/BUILD.gn

```diff
@@ -265,6 +266,15 @@ test("brave_unit_tests") {
     ]
   }
 
+  if (crypto_dot_com_enabled) {
+    sources += [
+      "//brave/components/crypto_dot_com/browser/crypto_dot_com_json_parser_unittest.cc"
```

Comment:
nit: As a follow up it's preferred to have a test dep, I'll link you to a PR for ipfs where I did something similar.